app/presenters/service_sign_in/choose_sign_in_presenter.rb

```diff
@@ -33,6 +33,12 @@ def back_link
       content_item['links']['parent'].first['base_path']
     end
 
+    def selected_option(selected_value)
+      symbolized_options.each do |option|
```

Comment:
Are we able to iterate through `options` and compare the `selected_value` against `value` as this already has the paramterized text value?

Reply:
Oh I've now seen the next commit where this is sorted out :)